src/main/java/org/openrewrite/java/migrate/lang/var/UseVarForPrimitive.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2021 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.migrate.lang.var;
+
+import lombok.EqualsAndHashCode;
+import lombok.Value;
+import org.jetbrains.annotations.NotNull;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Preconditions;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.config.RecipeExample;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.search.HasJavaVersion;
+import org.openrewrite.java.tree.Expression;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.JavaType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.lang.String.format;
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+
+@Value
+@EqualsAndHashCode(callSuper = false)
+public class UseVarForPrimitive extends Recipe {
+    @Override
+    public String getDisplayName() {
+        return "UseVarForPrimitives";
+    }
+
+    @Override
+    public String getDescription() {
+        //language=markdown
+        return "Try to apply local variable type inference `var` to primitiv variables where possible." +
+               "This recipe will not touch variable declaration with initializer containing ternary operators.";
+    }
+
+    @Override
+    public List<RecipeExample> getExamples() {
```

Comment:
You are typically not expected to override this method. Instead declare a corresponding test and annotate it with `@DocumentExample`.

Reply:
Tim also mentioned this, done ;)